.circleci/config.yml

```diff
@@ -128,6 +130,18 @@ jobs:
           command: |
             ./tools/third-party-licenses-check.sh
 
+  build:
+    docker:
+      - image: circleci/python:3.7
```

Comment:
I've used circleci's Python image here, the only thing we actually need is docker and git which is available on this image.